### ISSUE TITLE
[Bug]: Homebase 3 alarm armed event is incorrect

### ISSUE THREAD

**Issue by Palmke** [bropat/eufy-security-client#236]
### Client version

2.2.0

### Node version

14.18.0

### Operating System type

Docker

### Operating system version

Raspberry OS

### Describe the bug

The `alarm armed` event is incorrect in Homebase 3.

Whenver a mode switch occurs, `CMD_SET_TONE_FILE` is seen in the p2p session data.
This - for HB2 - is always seen when the alarm is armed. 
However, for HB3, `CMD_GET_DELAY_ALARM` is seen when the alarm is actually armed.

This funnily enough corresponds to, probably my own, comment in the p2p session code:
```
//When the alarm is armed, CMD_GET_DELAY_ALARM is called with event data 0, so ignore it
```

### To reproduce

View logging, switch mode and observe `alarm armed` emitted.
When switching to an alarm mode, observe `CMD_GET_DELAY_ALARM` is printed in the log.

I'm probably going to try to fix this myself, so this issue is just for administrative purposes.
No action needed from @bropat.

**Comment by Palmke** [comment]
It appears Homebase 3 has a bug in its firmware. 
Whenever a Homebase 2's alarm delay is cancelled, an `alarm armed` event is emitted and you can trigger a sensor again.
With Homebase 3 however, cancelling the alarm delay has some faulty behaviour where you cannot start another alarm delay. After <alarm delay> time (45s in my case), the alarm is activated. 

HB2:
```
Event[type=GUARD_MODE_CHANGED,payload={mode=AWAY, serial_number=T8010PZZZZZZZZ}]
Event[type=ALARM_ARM_DELAY_STARTED,payload={delay=15, serial_number=T8010PZZZZZZZZ}]
Event[type=ALARM_ARMED,payload={serial_number=T8010PZZZZZZZZ}]
Event[type=ALARM_DELAY_STARTED,payload={delay=45, cause=MOTION_SENSOR, serial_number=T8010PZZZZZZZZ}]
Event[type=ALARM_ARMED,payload={serial_number=T8010PZZZZZZZZ}]
Event[type=ALARM_DEACTIVATED,payload={cause=HUB_STOP_BY_APP, serial_number=T8010PZZZZZZZZ}]
```
HB3:
```
Event[type=GUARD_MODE_CHANGED,payload={mode=AWAY, serial_number=T8030PZZZZZZZZ}]
Event[type=ALARM_ARM_DELAY_STARTED,payload={delay=15, serial_number=T8030PZZZZZZZZ}]
Event[type=ALARM_ARMED,payload={serial_number=T8030PZZZZZZZZ}]
Event[type=ALARM_DELAY_STARTED,payload={delay=45, cause=MOTION_SENSOR, serial_number=T8030PZZZZZZZZ}]
Event[type=ALARM_DEACTIVATED,payload={cause=HUB_STOP_BY_APP, serial_number=T8030PZZZZZZZZ}]
```

**Comment by bropat** [comment]
Fixed in version `2.2.1`.
